research/sentiment_analysis/data/imdb.py

```diff
@@ -0,0 +1,44 @@
+from data.util import pad_sentence, to_dataset, START_CHAR, OOV_CHAR
+import tensorflow as tf, numpy as np
+
+NUM_CLASS = 2
+
+
+def construct_input_fns(vocabulary_size, sentence_length,
+                        batch_size, repeat=1):
```

Comment:
This might be able to fit in previous line. Make sure your IDE is set to have line limit 80.

Reply:
this line is 81